setup.py

```diff
@@ -23,7 +23,7 @@ def readme():
       packages=['active_subspaces', 'active_subspaces.utils'],
       install_requires=[
           'numpy',
-          'scipy >= 0.15.0',
+          'scipy >= 0.15.0,<=1.2.2',
           'matplotlib'
```

Comment:
I'll actually try to resolve this in `master` and let you merge master into this branch so you don't have to worry about it. We just need a "passing status" on this PR and there are a few ways to get there...